tests-requirements.txt

```diff
@@ -32,6 +32,9 @@ whisper
 # We could use Django<2.0 but we match what graphite requires instead.
 Django>=1.8,<1.11.99
 
+# Tracing
+opencensus
```

Comment:
I'd like to understand your plans.

Currently:

* it makes tracing disabled in "real live" (dev machine, staging, prod)
* it enables tracing in tests

Right?

Further in time, how to you plan to be able to enable / disable tracing?

Reply:
To tracer you need to create a tracer somewhere. It can be done in different way but you can start it from the middleware of django. So if you install opencensus and integrate it in django (graphite web) tracing will be enabled. I will PR documentation on how to use it in a next PR.